README.md

```diff
@@ -1,47 +1,66 @@
-`wrhs` CLI for Warehouse.ai
-====
+# `wrhs`
 
-Flawless CLI management of build pipelines & asset deployments for your front-end apps powered by [Warehouse.ai].
```

Comment:
🙏 We can make mistakes again! 🙏

Reply:
😁 it read a little bit weird. Don't mind to bring it back but would need somewhat different wording